services/ptrarchive_test.go

```diff
@@ -17,3 +19,25 @@ func TestPTRArchive(t *testing.T) {
 		t.Errorf("Found %d names, expected at least %d instead", result, expectedTest)
 	}
 }
+
+func TestPTRArchiveEndpoint(t *testing.T) {
+	if *networkTest == false {
+		return
+	}
+
+	var endpoint string
+	var ptr *PTRArchive
+	var err error
+
+
+	ptr = NewPTRArchive(testSystem)
+	endpoint = ptr.getURL(domainTest)
+
+	_, err = http.RequestWebPage(endpoint, nil, nil, "", "")
+
+	if err != nil {
+		if err.Error() == "404 Not Found" {
```

Comment:
I think this is fine for now. At some point we should look at the new error wrapping and convenience functions (`errors.Is` and `errors.As`). I wouldn't want to add something half-baked right now to make this particular function's errors nicer to work with when we could be on the verge of overhauling error handling throughout the repo.

Reply:
Ok, agreed.